subprojects/groovy-json/src/main/java/groovy/json/JsonOutput.java

```diff
@@ -492,7 +171,7 @@ private static void writeIterator(Iterator<?> iterator, CharBuf buffer) {
      */
     public static String prettyPrint(String jsonPayload) {
         int indentSize = 0;
-        // Just a guess that the pretty view will take 20 percent more than original.
+        // Just a guess that the pretty view will take a 20 percent more than original.
```

Comment:
Thanks for pointing that out I'll remove the `a`.  I missed that when fixing up the merge conflict caused by commit 7e8211fa314526ce.